### PR TITLE
feat: isort + poe

### PR DESCRIPTION
**Pull request by riomus**

### CODE REVIEW

**Comment by albert097** [comment]
I do not see isort check in run actions.

**Review by albert097 — CHANGES_REQUESTED** [review]
I was wrong about ability of isort to check unused imports. pyflakes and autoflake is able to do that.

https://github.com/PyCQA/pyflakes 
https://github.com/myint/autoflake

however, some people say that is not safe, but imo any setup that is not clearly visible is not a good practice. It is always possible to add `# ignore: something` is it is necessary.  

https://github.com/psf/black/issues/86
https://stackoverflow.com/questions/57275978/example-of-remove-unused-imports-can-have-bad-side-effects-in-python

File: .github/workflows/python_poetry_main.yml [hunk 1]
```diff
@@ -49,6 +49,8 @@ jobs:
         run: poetry run black --check ./
       - name: Check types
         run: poetry run mypy --strict .
+      - name: Check imports
+        run: poetry run isort **/*.py -c -v
```
Comment (albert097) [inline .github/workflows/python_poetry_main.yml:53]: I was wrong about ability of isort to check unused imports. pyflakes and autoflake is able to do that.

https://github.com/PyCQA/pyflakes
https://github.com/myint/autoflake

however, some people say that is not safe, but imo any setup that is not clearly visible is not a good practice. It is always possible to add # ignore: something is it is necessary.

psf/black#86
https://stackoverflow.com/questions/57275978/example-of-remove-unused-imports-can-have-bad-side-effects-in-python

Comment (riomus) [reply .github/workflows/python_poetry_main.yml:53]: will add it in separate mr

**Review by laugustyniak — APPROVED** [review]

**Review by laugustyniak — APPROVED** [review]

**Review by laugustyniak — DISMISSED** [review]

**Review by albert097 — APPROVED** [review]

**Review by ktagowski — APPROVED** [review]

**Pull request merged**